### PR TITLE
[WIP] Attach node to stripe on secured cluster.

### PR DESCRIPTION
**Pull request by prasantakumar82**

### CODE REVIEW

**Comment by prasantakumar82** [comment]
> Agree with the impl, but there are many more occurrences of calls to `fetchDiagnosticService()` to replace...

Do we need to follow unsecured path for other use cases? For example with diagnostic command, if user passing "security root directory" and if the security is not enabled in cluster, should the request fail instead of falling back to non-secured connection.

File: dynamic-config/cli/config-tool/src/main/java/org/terracotta/dynamic_config/cli/config_tool/command/RemoteCommand.java [hunk 1]
```diff
@@ -431,14 +431,14 @@ protected final void ensureNodesAreEitherActiveOrPassive(Map<InetSocketAddress,
...
-    try (DiagnosticService diagnosticService = diagnosticServiceProvider.fetchDiagnosticService(expectedOnlineNode)) {
+    try (DiagnosticService diagnosticService = diagnosticServiceProvider.fetchDiagnosticServiceWithFallback(expectedOnlineNode)) {
       return diagnosticService.getProxy(TopologyService.class).isActivated();
     }
   }
 
   protected final void resetAndStop(InetSocketAddress expectedOnlineNode) {
     logger.info("Reset node: {}. Node will stop in 5 seconds", expectedOnlineNode);
-    try (DiagnosticService diagnosticService = diagnosticServiceProvider.fetchDiagnosticService(expectedOnlineNode)) {
+    try (DiagnosticService diagnosticService = diagnosticServiceProvider.fetchDiagnosticServiceWithFallback(expectedOnlineNode)) {
```
Comment (mathieucarbou) [inline dynamic-config/cli/config-tool/src/main/java/org/terracotta/dynamic_config/cli/config_tool/command/RemoteCommand.java:441]: not needed

**Comment by mathieucarbou** [comment]
> > Agree with the impl, but there are many more occurrences of calls to `fetchDiagnosticService()` to replace...
> 
> Do we need to follow unsecured path for other use cases? For example with diagnostic command, if user passing "security root directory" and if the security is not enabled in cluster, should the request fail instead of falling back to non-secured connection.

oh yes just the attach case with a mix of secured / non secured nodes is enough.

**Comment by saurabhagas** [comment]
I have a few concerns about this PR. Let's huddle together briefly today and discuss this change.

**Comment by mathieucarbou** [comment]
@prasantakumar82 : converting this PR to draft. This is not finished (see my comment) and requires to be tested in EE with a system test. Thanks!

**Pull request closed without merging**